test/foundry/unit/concrete/erc4337account/TestERC4337Account_OnlyEntryPointOrSelf.t.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "../../../utils/Imports.sol";
+import "../../../utils/SmartAccountTestLab.t.sol";
+// import {UserOperation} from "path/to/UserOperation.sol"; // Update this path
+
+contract TestERC4337Account_ValidateUserOp is Test, SmartAccountTestLab {
+    ERC4337Account public account;
+    MockValidator public validator;
+    address public userAddress;
+
+    function setUp() public {
+        init();
+        userAddress = address(BOB.addr);
+        validator = new MockValidator();
+    }
+
+    function test_ValidateUserOp_ValidOperation() public {
```

Comment:
these tests are not really related to access control onlyEntryPointOrSelf but sig authorization

Reply:
yes actually, that's a mistake, I duplicated `TestERC4337Account_ValidateUserOp.t.sol` to edit it, It was not supposed to be pushed 😅